crates/connectors/ndc-postgres/src/state.rs

```diff
@@ -79,7 +79,25 @@ async fn create_pool(
 ) -> Result<PgPool, InitializationError> {
     let connect_options = PgConnectOptions::from_url(connection_url)
         .map_err(InitializationError::UnableToCreatePool)?;
-    PgPoolOptions::new()
+
+    let pool_options = match pool_settings.check_connection_after_idle {
+        // The default behaviour is to always ping on acquire.
```

Comment:
Actually, the default seems to be to check after 60 seconds.

Reply:
Sorry, I'll rephrase, the default *sqlx* behaviour is to always check :)